ui/src/components/ui/settings-tabs/general-tab/alert-message-settings/alert-message-settings.tsx

```diff
@@ -0,0 +1,60 @@
+import { observer } from 'mobx-react-lite'
+import { useInjection } from 'inversify-react'
+import { useTranslation } from 'react-i18next'
+import { Icon20InfoCircleOutline } from '@vkontakte/icons'
+import { FormItem, FormLayoutGroup, Input, SelectionControl, Spacing, Switch } from '@vkontakte/vkui'
+
+import { BaseSelect } from '@/components/lib/base-select'
+import { ContentCard } from '@/components/lib/content-card'
+
+import { SettingsService } from '@/services/settings-service/settings-service'
+
+import { CONTAINER_IDS } from '@/config/inversify/container-ids'
+
+import styles from './alert-message-settings.module.css'
```

Comment:
Styling a single component seems redundant. Does it really need to be `width: fit-content`? Maybe we are using VKUI not in the way that is expected of us? Can we leave it to be in the resemblance of the VKIU style guide?